README.md

```diff
@@ -38,17 +38,17 @@ To facilitate the installation process, a script is provided to install/update d
 
 This process will create an environment with all the necessary dependencies. To activate the environment run:
 ```
-source venv/bin/activate 
+source venv/bin/activate
 ```
 
 ## Getting started
-Examples can be found in [`examples`](./examples). [`examples/README.md`](./examples/README.md) provides an overview 
-of the different examples in this repo. 
+Examples can be found in [`examples`](./examples). [`examples/README.md`](./examples/README.md) provides an overview
+of the different examples in this repo.
 
 ## Do you want to contribute?
 
-If you would like to contribute, please review our [code of conduct](./CODE_OF_CONDUCT.md) and 
-[contributing](./CONTRIBUTING.md) document.
+If you would like to contribute, please review our [code of conduct](./.github/CODE_OF_CONDUCT.md) and
+[contributing](./.github/CONTRIBUTING.md) document.
 
 ## License
 
```

Comment:
Replace the next section with:

This project is licensed under the terms of the MIT License. See license.txt for details.